templates/emails/email-styles.php

```diff
@@ -11,25 +11,28 @@
  * @version     1.31.0
  */
 
-if ( ! defined( 'ABSPATH' ) ) {
+if ( ! defined( "ABSPATH" ) ) {
```

Comment:
Why change all single quotes in the file to double-quotes ?
Usually it's always single quotes, except if you need to interpret a var in the string, but maybe there's a reason here ? (I think it might just be something your IDE did by itself)

Reply:
Oops that might have been a find-n-replace gone wrong